### PR TITLE
[chrome] Add missing type to chrome.runtime

### PR DESCRIPTION
**Pull request by mionim**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [type-RequestUpdateCheckStatus](https://developers.chrome.com/extensions/runtime#type-RequestUpdateCheckStatus)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -5242,6 +5242,9 @@ declare namespace chrome.runtime {
         version: string;
     }
 
+    /** Result of the update check. */
+    export type RequestUpdateCheckStatus = 'throttled' | 'no_update' | 'update_available';
```
Comment (matcarlson) [inline types/chrome/index.d.ts]:
```suggestion
    export interface RequestUpdateCheckStatus = 'throttled' | 'no_update' | 'update_available';
```

**Review by matcarlson — CHANGES_REQUESTED** [review]
Could you modify `type` to `interface` to match the rest of the file?

Checked source, LGTM otherwise.

File: types/chrome/index.d.ts (hunk 1 above)
Comment (MiOnim) [reply types/chrome/index.d.ts]: The suggested change results in syntax error. This is not an interface. A variable with this new type has to be a `string`, not an `object`.

Also, there are 4 more instances of `type` in this file.

Comment (matcarlson) [reply types/chrome/index.d.ts]: Makes sense. Thanks for the clarification.

**Review by matcarlson — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**